blockball-bukkit-plugin/src/main/java/com/github/shynixn/blockball/bukkit/logic/business/listener/BallListener.kt

```diff
@@ -251,13 +267,13 @@ class BallListener @Inject constructor(
     }
 
     /**
-     * Gets called when a player tries to leah a ball and cancels all of it.
+     * Gets called when a player tries to leash a ball and cancels all of it.
      *
      * @param event event
      */
     @EventHandler
     fun entityLeashEvent(event: PlayerLeashEntityEvent) {
-        if (event is LivingEntity) {
+        if (event.entity is LivingEntity) {
```

Comment:
Unittesting listeners would be a good idea. But I'll accept the PR without them too.

Reply:
I dunno what is unittesting tbh so I'm afraid I can't deal with it

Reply:
Maybe I'll explain it to you in the future :)